packages/database/prisma/schema.prisma

```diff
@@ -23,7 +23,10 @@ model Comment {
 model LessonProgress {
   id            String    @id @default(uuid())
   userId        String
-  lessonSlug    String
+  lessonId      String?
+  sectionId     String?
+  moduleId      String?
```

Comment:
the idea here is that if we store the module/section with the lesson progress we could load all progress for a module easily. This uses ID not slug.